lib/pages/my_reports_pages/components/reports_list_widget.dart

```diff
@@ -230,8 +230,38 @@ class _MyReportsListState extends State<ReportsList> {
                     SizedBox(
                       height: 20,
                     ),
-                    Style.titleMedium(
-                      Utils.getTranslatedReportType(context, report.type)),
+                    Row(
+                      mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                      children: [
+                        Text(
+                          'Title',
```

Comment:
`Utils.getTranslatedReportType(context, report.type))`

Reply:
I've added it back, I just changed it for debugging